esmvaltool/recipes/recipe_ocean_heatcontent.yml

```diff
@@ -0,0 +1,100 @@
+# ESMValTool
+# recipe_ocean_example.yml
+---
+documentation:
+  description: |
+    Recipe to demonstrate several simple plots based on the monthly ocean
+    temperature. Please use this file as a template for adding additional
+    fields into the ocean. This work based on the BGC-val toolkit GMD-2018-103.
+
+  authors:
+    - demo_le
+
+  maintainer:
+    - schl_ma
+
+  references:
+    - demora2018gmd
+
+  projects:
+    - ukesm
+
+
+datasets:
+  # working datasets
+  #OO - {dataset: CanESM2, project: CMIP5, mip: Omon, exp: historical, ensemble: r1i1p1, start_year: 1970, end_year: 2004}
```

Comment:
Why are all these datasets commented out?

Reply:
Because they are currently not available on JASMIN due to an unintended data loss.

Reply:
The pull request shouid not  be merged with commented out stuff in it. Either the lines should be completely removed, or uncommented (but then it should also be tested that it actually works).